internal/protoschema/jsonschema/jsonschema.go

```diff
@@ -170,7 +170,13 @@ func (p *jsonSchemaGenerator) generateDefault(desc protoreflect.MessageDescripto
 func (p *jsonSchemaGenerator) setDescription(desc protoreflect.Descriptor, schema map[string]any) {
 	src := desc.ParentFile().SourceLocations().ByDescriptor(desc)
 	if src.LeadingComments != "" {
-		schema["description"] = strings.TrimSpace(src.LeadingComments)
+		comments := strings.TrimSpace(src.LeadingComments)
+		if parts := strings.SplitN(comments, "\n\n", 2); len(parts) >= 2 {
```

Comment:
Do you want to do this even it it's only one element? Should also document somewhere (in code, in whatever attached doc we have) that this is happening

Reply:
Good point! Done.

Reply:
Where is the documentation? I don't see any code comments or otherwise

Reply:
/face_palm

Reply:
(code comments added)